infrastructure-playbooks/purge-cluster.yml

```diff
@@ -408,6 +400,14 @@
      ceph_data_partlabels.rc == 0 and
      zap_block_devs
 
+  - name: get ceph journal partitions
+    shell: |
+      blkid | awk '/ceph journal/ { sub (":", "", $1); print $1 }'
+    when:
+      - ceph_journal_partlabels.rc == 0
```

Comment:
With a single line, we usually do it like this: `when: ceph_journal_partlabels.rc == 0` only.
(yes I know, **I** did that...)